numpy/random/_pickle.py

```diff
@@ -14,14 +14,16 @@
                  }
 
 
-def __bit_generator_ctor(bit_generator_name='MT19937'):
+def __bit_generator_ctor(bit_generator_name='MT19937', seed_seq=None):
     """
     Pickling helper function that returns a bit generator object
 
     Parameters
     ----------
     bit_generator_name : str
         String containing the name of the BitGenerator
+    seed_seq : tuple
```

Comment:
```suggestion
    seed_seq : SeedSequence or None
```
not that it matters really

Reply:
Fixed (ISeedSequence or None) for generality.